archivers/pigz/Portfile

```diff
@@ -19,13 +19,12 @@ long_description    pigz, which stands for parallel implementation of gzip, \
 homepage            http://www.zlib.net/pigz/
 master_sites        ${homepage}
 
-checksums           rmd160  99ffd6ca6acbf29c436d80e814a144dfd1bd3337 \
-                    sha256  a4f816222a7b4269bd232680590b579ccc72591f1bb5adafcd7208ca77e14f73
+checksums           rmd160  42bd149f6a639d7db8517c23f3f0ccf6b0d7524a \
+                    sha256  2eed7b0d7449d1d70903f2a62cd6005d262eb3a8c9e98687bc8cbb5809db2a7d \
+                    size    106840
 
 depends_lib         port:zlib
 
-patchfiles          patch-Makefile.diff
```

Comment:
If you look at the build log, it now uses `gcc`, indicating that this change — or something similar — remains required.

Reply:
Updated the patch file and can see it now uses clang.